Resources/Locale/en-US/cargo/cargo-pallet-console-component.ftl

```diff
@@ -0,0 +1,7 @@
+
+# Cargo pallet sale console
+cargo-pallet-console-menu-title = Cargo sale console
+cargo-pallet-menu-appraisal-label = Estimated Value:{" "}
+cargo-pallet-menu-count-label = Number of sale items:{" "}
```

Comment:
if this works put an updated screenshot in the pr body

Reply:
Yea thats how I've seen it in the other uis.  It was already localized I just didn't see it.  I'll update screenshots and hopefully address other reviews after work tonight if I don't immediately fall asleep like I have been